### PR TITLE
Update cluster resource loading states

### PR DESCRIPTION
**Pull request by maciaszczykm**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes https://github.com/kubermatic/dashboard/issues/2609

**Special notes for your reviewer**:
At the moment it is added only for cluster resources as mentioned in the issue. @cschieder if we want to use it in more places please let us know about them in a separate issue.

Zrzut ekranu 2020-10-5 o 12 23 49
Zrzut ekranu 2020-10-5 o 12 23 38

**Release note**:

```release-note
Updated cluster resource loading states.
```

### CODE REVIEW

File: src/app/cluster/cluster-details/rbac/rbac.component.html [hunk 1]
```diff
@@ -99,26 +99,21 @@ <h2>Cluster</h2>
...
-      <div class="km-row"
-           *ngIf="!isClusterRunning">
-        <mat-spinner color="accent"
-                     class="km-spinner"
-                     [diameter]="25"></mat-spinner>
-      </div>
-    </ng-container>
+    <div class="km-row km-empty-list-msg"
+         *ngIf="hasNoData(clusterBindings)">
+      No machine deployments available.
```
Comment (floreks) [inline src/app/cluster/cluster-details/rbac/rbac.component.html:104]: Machine deployments?

Comment (maciaszczykm) [reply src/app/cluster/cluster-details/rbac/rbac.component.html]: Copy paste error :) It's fixed now.

**Pull request merged**